repos/system_upgrade/el8toel9/actors/nisscanner/libraries/nisscan.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from leapp.libraries.stdlib import api
+from leapp.models import NISConfig
+
+PACKAGES_NAMES = ('ypserv', 'ypbind')
+YPBIND_CONF_FILE = '/etc/yp.conf'
+YPSERV_DIR_PATH = '/var/yp'
+YPSERV_DEFAULT_FILES = ('binding', 'Makefile', 'nicknames')
```

Comment:
I mean I do not know anything about `ypserv` but there's a Makefile?

Reply:
Yes, ypserv is an old package. Makefile is used, among other things, to propagate a new maps to slaves.

Reply:
Ok thanks for clarifying. It felt strange.